sqlmesh/dbt/manifest.py

```diff
@@ -371,6 +374,24 @@ def _extra_dependencies(self, target: str) -> Dependencies:
                 args = [_jinja_call_arg_name(arg) for arg in node.args]
                 if args and args[0]:
                     dependencies.variables.add(args[0])
+            elif len(call_name) == 1:
+                macro_name = call_name[0]
+                if macro_name in {**BUILTIN_GLOBALS, **BUILTIN_FILTERS}:
```

Comment:
This dict will be created for each macro.

Reply:
When it's a very small dict (~20), does it impact performance? Originally I checked in both separately, but thought this was more readable.

Reply:
make a new constant and use that here

Reply:
depending on how many times this is called, it could have a significant impact on performance